Tesla.css

```diff
@@ -0,0 +1,478 @@
+* {
+  margin: 0;
+}
+
+body {
+  max-width: 1920px;
+  margin: 0 auto;
+  font-family: Montserrat, sans-serif;
+  font-weight: 600;
+  font-style: normal;
+}
+
+.section_first {
+  position: relative;
+}
+
+.logo {
+  width: 166px;
+}
+
+.link {
+  text-decoration: none;
+  color: #000;
+}
+
+.header {
+  padding: 24px 60px 0px 42px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+ 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Tesla-n/pullRequest?prid=5749910)